src/radix_connect/wallet_interaction/dapp_wallet_interaction/wallet_to_dapp/response.rs

```diff
@@ -23,6 +23,15 @@ impl HasSampleValues for WalletToDappInteractionResponse {
     }
 }
 
+impl WalletToDappInteractionResponse {
+    pub fn is_success(&self) -> bool {
+        match self {
```

Comment:
have you run `cargo clippy --fix --allow-staged` ? if not, do it :) I think it will change this to `matches!` macro usage!

Reply:
no it didn't change it, kept it as is :)